cmd/validate.go

```diff
@@ -0,0 +1,69 @@
+package cmd
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+
+	"github.com/go-gota/gota/dataframe"
+	"github.com/spf13/cobra"
+)
+
+var validateCmd = &cobra.Command{Use: "validate", Short: "Validate ros related data"}
+
+var validateCSV = &cobra.Command{
+	Use:   "csv [input csv file path]",
+	Short: "Validate ros-usage CSV file",
+	Long:  "Validate ros-usage CSV file",
+	Run: func(cmd *cobra.Command, args []string) {
+		input_file := args[0]
+		if _, err := os.Stat(input_file); os.IsNotExist(err) {
+			fmt.Printf("CSV file: %s does not exist\n", input_file)
+			os.Exit(1)
+		}
+		f, err := os.Open(input_file)
+		if err != nil {
+			panic(err.Error())
+		}
+		defer f.Close()
+
+		csv := csv.NewReader(f)
+		records, err := csv.ReadAll()
+		if err != nil {
+			panic(err.Error())
+		}
+
+		df := dataframe.LoadRecords(records)
+		groups := df.GroupBy("container_name", "pod", "owner_name", "owner_kind", "workload", "workload_type", "namespace", "image_name").GetGroups()
+
+		// for _, v := range groups {
+		// 	if v.Nrow() > 26 {
+		// 		sorted := v.Arrange(
+		// 			dataframe.Sort("interval_start"),
+		// 		)
+		// 		for _, data := range sorted.Maps() {
+		// 			asd, _ := utils.ConvertStringToTime(data["interval_start"].(string))
+		// 			fmt.Println(asd)
+		// 		}
+		// 	}
+		// }
```

Comment:
@patilsuraj767  - Though the pull-request is merged, are these unwanted leftovers?
In case intentional, as nitpick could you add parent comment with keyword (example - //TODO or //FIXME) that explains reason behind keeping this code. Otherwise, does it make sense to remove these lines?

Reply:
@kgaikwad that was not intentional. I was something else on which I am working on. I have created the PR to remove that file - https://github.com/RedHatInsights/ros-ocp-backend/pull/75